ITRStartTask/StoriesView/Presenter/StoriesPresenter.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  StoriesPresnterImplementation.swift
+//  ITRStartTask
+//
+//  Created by Homyakov, Ilya2 on 20/08/2018.
+//  Copyright © 2018 Homyakov, Ilya2. All rights reserved.
+//
+
+import UIKit
+
+class StoriesPresenter: StoriesPresenterProtocol {
+    unowned let view: StoriesViewProtocol
+    private var storiesDataProvider: StoriesDataProviderProtocol
+    fileprivate var storyItems: [StoryItem]
+
+    init(view: StoriesViewProtocol, storiesDataProvider: StoriesDataProviderProtocol) {
+        self.view = view
+        self.storiesDataProvider = storiesDataProvider
+        self.storyItems = []
+    }
+
+    func show() {
+        if storyItems.count > 0 {
+            return
```

Comment:
Presenter should guarantee data displaying

Reply:
Fixed